README.md

```diff
@@ -66,40 +66,13 @@ To enable message contents to be included in the traces, set the following envir
-```python
-from .api.search_index_manager import SearchIndexManager
-
-search_index_manager = SearchIndexManager (
-    endpoint=self.search_endpoint,
-    credential=your_credentials,
-    index_name=index_name,
-    dimensions=100,
-    model="text-embedding-3-small",
-    embeddings_client=embedding_client,
-)
-await search_index_manager.build_embeddings_file(
-    input_directory='data',
-    output_file='data/embeddings.csv'
-    sentences_per_embedding=4
-)
-```
-Please note the `sentences_per_embedding  parameter`, which specifies the number of sentences used to construct the embedding. The larger this number, the broader the context that will be identified during the similarity search.
-For each question asked from the application, we first search the answer in vector store and if the answer was found, we return the response based on data provided in the data set. To deploy the application with RAG model, please provide the next environment variables:
-```
-USE_SEARCH_SERVICE=true
-AZURE_AI_SEARCH_INDEX_NAME=index_sample
-AZURE_AI_EMBED_DEPLOYMENT_NAME=text-embedding-3-small
-```
-If these variables, except `USE_SEARCH_SERVICE`, which is `true` by default, were not set, or there is no Azure AI Search connection, RAG search will not be used. 
-In this application we are creating index, called `index_sample` in Azure Search Service. The index can be created by following [instructions](https://learn.microsoft.com/azure/ai-services/agents/how-to/tools/azure-ai-search?tabs=azurecli%2Cpython&pivots=overview-azure-ai-search), or by calling the next code:
-```python
-search_index_manager.ensure_index_created(vector_index_dimensions)
-search_index_manager.upload_documents(embeddings_path)
-```
-In this case `vector_index_dimensions` needs to be provided only if `dimensions` were not set during `SearchIndexManager` object creation. If the index will be present before the aplication deployment, application will skip document upload and will use the existing index.
+## Retrieval-Augmented Generation (RAG)
+ 
+The Retrieval-Augmented Generation (RAG) feature helps improve the responses from your application by combining the power of large language models (LLMs) with extra context retrieved from an external data source. Simply put, when you ask a question, the application first searches through a set of relevant documents (stored as embeddings) and then uses this context to provide a more accurate and relevant response. If no relevant context is found, the application returns the LLM response directly.
+ 
+To enable and configure the RAG feature in your application, please refer to the following detailed documentation:
```

Comment:
Here we could mention if the feature is enabled by default. It was enabled by default, right?
But then for detailed configuration, please refer the guide below

Reply:
Fixed and moved RAG.md to docs folder